### PR TITLE
DCS-1387 update dependencies

### PR DESCRIPTION
**Pull request by GurnankCheema**

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -19,18 +19,21 @@ dependencies {
...
   runtimeOnly("org.postgresql:postgresql:42.3.1")
 
   testImplementation("io.jsonwebtoken:jjwt:0.9.1")
   testImplementation("com.github.tomakehurst:wiremock-standalone:2.27.2")
   testImplementation("org.springframework.security:spring-security-test")
-  testImplementation("io.mockk:mockk:1.12.0")
+  testImplementation("io.mockk:mockk:1.12.2")
+}
+repositories {
+  mavenCentral()
```
Comment (petergphillips) [inline build.gradle.kts:36]: why is this required?

Comment (GurnankCheema) [reply build.gradle.kts]: removed

**Review by petergphillips — APPROVED** [review]
LGTM
